web/viewer.css

```
@@ -1430,6 +1345,144 @@ dialog :link {
   z-index: 50000; /* should be higher than anything else in PDF.js! */
 }
 
+#findbar {
+  background-color: var(--toolbar-bg-color);
+  cursor: default;
+  font: message-box;
+  font-size: 12px;
+  height: auto;
+  inset-inline-start: 64px;
+  line-height: 14px;
+  margin: 4px 2px;
+  min-width: 300px;
+  padding: 0 4px;
+  position: absolute;
+  text-align: left;
+  top: var(--toolbar-height);
+  z-index: 30000;
+
+  * {
+    float: var(--inline-start);
+    position: relative;
+  }
+
+  > div {
+    height: var(--toolbar-height);
+  }
+
+  button {
+    font: message-box;
+    outline: none;
+  }
+
+  input {
+    font: message-box;
+    outline: none;
+
+    &:focus-visible + label {
+      background-color: var(--button-hover-color);
+      color: var(--toggled-btn-color);
+    }
+
+    &[type="checkbox"] {
+      pointer-events: none;
+
+      &:checked + .toolbarLabel {
+        background-color: var(--toggled-btn-bg-color) !important;
+        color: var(--toggled-btn-color);
+      }
+    }
+  }
+
+  label {
+    user-select: none;
+
+    &:hover {
+      background-color: var(--button-hover-color);
+      color: var(--toggled-btn-color);
+    }
+  }
+
+  #findbarInputContainer {
+    margin-inline-end: 4px;
+
+    #findInput {
+      width: 200px;
+
+      /*#if !MOZCENTRAL*/
+      &::-webkit-input-placeholder {
+        color: rgb(191 191 191);
+      }
+
+      /*#endif*/
```

Nit: This would be a tiny bit easier to read at a glance by having the `endif` *immediately* after the end of the block (i.e. no blank line in between).
```suggestion
      /*#if !MOZCENTRAL*/
      &::-webkit-input-placeholder {
        color: rgb(191 191 191);
      }
      /*#endif*/
      
```

---

Fixed in the new commit. It's more consistent with how it's now; it was mainly done like this to make sure that the generated output looked correct w.r.t. newlines, but thinking about this again I also care more about the readability of the source file than the generated output.